src/Microsoft.AspNet.JsonPatch/Exceptions/JsonPatchException.cs

```diff
@@ -3,14 +3,13 @@
 
 using System;
 using Microsoft.AspNet.JsonPatch.Operations;
-using Microsoft.Framework.Internal;
 
 namespace Microsoft.AspNet.JsonPatch.Exceptions
 {
-    public class JsonPatchException<TModel> : JsonPatchException where TModel : class
+    public class JsonPatchException : JsonPatchExceptionBase //where TModel : class
```

Comment:
Used it to be able to set the InnerException - but it's better without the custom base class by calling : base(message, innerException).  So you're right, we don't need that base class.  Will change.